Examples/UIComponentExample/Examples/Complex layout/Components/Cell.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Cell.swift
+//  UIComponentExample
+//
+//  Created by y H on 2021/7/21.
+//  Copyright © 2021 Luke Zhao. All rights reserved.
+//
+
+import UIComponent
+import UIKit.UIScreen
+
+struct Cell: ComponentBuilder {
```

Comment:
the name `Cell` is too generic. Let's give it a more identifiable name.

Reply:
✅